### PR TITLE
posix: Improve phrasing and formatting

### PR DESCRIPTION
**Pull request by antisaling**

### CODE REVIEW

**Comment by antisaling** [comment]
Hey there, excellent tutorial! As I'm following it for an assignment I thought I'd come by and make a few suggestions. I can keep them all in this one PR if you like but I don't know how many things I'll spot.

**Review by IanLee1521 — APPROVED** [review]
Looks good to me thank you! Keep them coming!

**Comment by IanLee1521** [comment]
I see you have this as a draft, once you're ready for it to merge, feel free to remove the draft marker and I can merge it.

File: posix/joining_and_detaching.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
 ![joining](images/joining.gif)
 
-The `pthread_join()` subroutine blocks the calling thread until the specified threadid thread terminates.
+The `pthread_join()` subroutine blocks the calling thread until the specified `thread` terminates.
 
-The programmer is able to obtain the target thread's termination return `status` if it was specified in the target thread's call to `pthread_exit()`.
+The programmer is able to obtain the target thread's termination `status` if it was specified in the target thread's call to `pthread_exit()`.
 
-A joining thread can match one `pthread_join()` call. It is a logical error to attempt multiple joins on the same thread.
+A thread be only joined once. It is a logical error to attempt multiple joins on the same thread.
```
Comment (IanLee1521) [inline posix/joining_and_detaching.md]:
```suggestion
A thread can only be joined once. It is a logical error to attempt multiple joins on the same thread.
```

**Review by IanLee1521 — APPROVED** [review]
This looks great, thank you!

**Pull request merged**